src/main/java/com/garbagemule/MobArena/ArenaImpl.java

```diff
@@ -1468,6 +1469,16 @@ public AutoStartTimer getAutoStartTimer() {
         List<Player> result = new LinkedList<>();
         result.addAll(lobbyPlayers);
         result.removeAll(readyPlayers);
+        result.sort(Comparator.comparing(Player:: getName));
```

Comment:
I'm not sure the sorting should be in `ArenaImpl`. The reason `getNonreadyPlayers()` returns a List rather than a Set is likely because of its original call site in the `notready` command, where it is fed to a utility method that stringifies a List. So it's probably past laziness more than anything :P

I think it makes more sense to do the sorting at the caller site rather than in here. If nothing else, it reduces the blast radius of the pull request and keeps the changes "collected" in the signs/rendering code.